.github/workflows/ci.yml

```diff
@@ -119,23 +119,8 @@ jobs:
         run: |
           echo LD_LIBRARY_PATH=`pwd`/lib >> $GITHUB_ENV
 
-          case `uname` in
-            Linux)
-                wxPROC_COUNT=`nproc`
-                ;;
-
-            Darwin)
-                wxPROC_COUNT=`sysctl -n hw.ncpu`
-                ;;
-
-            *)
-                echo "*** Unknown platform: `uname` ***"
-                wxPROC_COUNT=0
-                ;;
-          esac
-
-          ((wxPROC_COUNT++))
-          echo wxPROC_COUNT=$wxPROC_COUNT >> $GITHUB_ENV
+          . build/tools/proc_count.sh
+          wxPROC_COUNT=`proc_count`
```

Comment:
This is pretty minor and maybe it's just me, but sourcing a script and then setting a variable using stdout redirection seems unusual to me. I'd either run the script, i.e.
```
wxPROC_COUNT=`./build/tools/proc_count.sh`
```
or source it to set the variable directly.

Reply:
Implemented your first suggestion.